File: module7/Collaboration/Python/AddUserToRoom.py
import requests

url = "https://api.ciscospark.com/v1/memberships"

payload = "{\n  \"roomId\" : \"Y2lzY29zcGFyazovL3VzL1JPT00vMzRjNTExZTAtNWIzYS0xMWU2LTlmNTgtYzcwMGY5YjM0YWFl\",\n  \"personEmail\" : \"twittbrod@gmail.com\",\n  \"isModerator\" : false\n}"
headers = {
    'authorization': "Bearer NTIyN2U5NGQtY2JjZi00NGYxLWE0OTMtYzMwYTY3OWQzMDljOGE0YzA3ZTctNjY1",
    'content-type': "application/json",
    'cache-control': "no-cache",
    'postman-token': "f253c29e-8987-1e05-976d-ceab2ff2f4de"
    }

response = requests.request("POST", url, data=payload, headers=headers)

print(response.text)

File: module7/Collaboration/NodeJS/AddUserToRoom.js
var request = require("request");

var options = { method: 'POST',
  url: 'https://api.ciscospark.com/v1/memberships',
  headers: 
   { 'postman-token': 'a05472ec-0f7f-3f31-6657-378aa9b453d1',
     'cache-control': 'no-cache',
     'content-type': 'application/json',
     authorization: 'Bearer NTIyN2U5NGQtY2JjZi00NGYxLWE0OTMtYzMwYTY3OWQzMDljOGE0YzA3ZTctNjY1' },
  body: 
   { roomId: 'Y2lzY29zcGFyazovL3VzL1JPT00vMzRjNTExZTAtNWIzYS0xMWU2LTlmNTgtYzcwMGY5YjM0YWFl',
     personEmail: 'twittbrod@gmail.com',
     isModerator: false },
  json: true };

request(options, function (error, response, body) {
  if (error) throw new Error(error);

  console.log(body);
});
